app/src/main/java/com/stevesoltys/seedvault/restore/RestoreViewModel.kt

```diff
@@ -200,19 +204,13 @@ internal class RestoreViewModel(
             return
         }
 
-        // we need to retrieve the restore sets before starting the restore
-        // otherwise restoreAll() won't work as they need the restore sets cached internally
-        val observer = RestoreObserver { observer ->
-            // this lambda gets executed after we got the restore sets
-            // now we can start the restore of all available packages
-            val restoreAllResult = session.restoreAll(token, observer, monitor)
```

Comment:
Are we sure that our list of packages is equivalent to this `restoreAll()` call? Nothing is missing?

Reply:
This I'm not sure of. But I figure we only can restore packages that are in the backup anyways?